data.py

```diff
@@ -93,54 +93,49 @@ def populate(oxigraph_url=None, write_to_db=True):
     # TODO - need to iterate all files in ./edtions not just the one
 
     # Load from disk
-    editions_source_dict = subbed_metadata_store_content_path / "editions"
-
+    editions_source_path = Path(
+        subbed_metadata_store_content_path / "editions"
+    )
     # Validate then add to graph
     schema = schemas.Editions
-    process_json_files(g, editions_source_dict, schema)
+    process_json_files(g, editions_source_path, schema)
 
     # ------------------
     # Versions resources
     # ------------------
 
     # TODO - need to iterate all files in ./editions/versions not just the one
-    versions_source_dict = Path(
+    versions_source_path = Path(
         subbed_metadata_store_content_path / "editions/versions"
     )
 
     # Validate then add to graph
     schema = schemas.Versions
-    process_json_files(g, versions_source_dict, schema)
+    process_json_files(g, versions_source_path, schema)
 
     # ------------------
     # Topics resources
     # ------------------
 
-    topics_source_path = Path(subbed_metadata_store_content_path / "topics.json")
-    with open(topics_source_path) as f:
-        topics_source_dict = json.load(f)
-
+    topics_source_path = Path(
```

Comment:
this pattern won't work for the individual files, its saying "look in a directory equal to /src/store/metadata/stub/content/topics ..." which isn't a directory so no json files are found. So `process_json_files` is running but not processing anything.

we need some assertions around this behaviour (silent fails are bad - so its on my list of cards to add) but for now can you make sure that the implementation for topic and publishers follows the pattern as used as main branch.